frontend/src/components/ReportSelect/fields.tsx

```diff
@@ -276,6 +276,355 @@ export const reportType = {
         },
       ],
     },
+    {
+      reportCategory: "divisional_project_reports",
+      value: "Tab_50_rpt_PF_NetRecoverySummaryByQuarter",
+      label: "Net Recovery Summary By Quarter",
+      reportParamCategory: [
+        {
+          field: fieldTypes.fiscal,
+          type: requestTypes.route,
+          isRequired: true,
+          hasXls: true,
+        },
+      ],
+    },
+    {
+      reportCategory: "divisional_project_financials",
```

Comment:
I'm a bit confused why we have two configurations... one in the playground/reportSelectorconfig.tx, and one here.
@ASpiteri-BCGov is this one needed,

Reply:
They aren't both needed...

Reply:
Yes it's interesting, the place that the ticket [indicated to add the reports](https://github.com/bcgov/gdx-agreements-tracker/blob/development/frontend/src/components/ReportSelect/fields.tsx#L74) was not sufficient for the new options, so they had to be added in reportSelectorConfig as well.

Maybe a mistake on the ticket?